### DISCUSSION TITLE
Declaring a transfer-function mit zpk and plotting time responses

### DISCUSSION THREAD

**Discussion by MEnzmann** [python-control/python-control#1061] [Q&A]
Hi all! 

I am little puzzled about the declaration of transfer-functions with control.zpk. The step-responses / impulse-responses of a transfer-function declared with control.zpk seem to unstable, although the poles are in the left-half plane and a bode plot does not show unstable poles. Am I missing something in the following MME?

Help would be appreciated! TIA!

_import control as ct
import control.matlab as ml
import matplotlib.pyplot as pl
#### Declare stable system as zpk 
Gzp = ct.zpk([-5],[-1,-10], gain=4)
print(Gzp)
#### Declare same system as tf
Gtf = ct.tf([4,20], [1,11, 10])
print(Gtf)
#### Yet: the Gzp system is instable in simulations while Gtf is stable (as expected)
[Tzp, Yzp] = ct.impulse_response(Gzp)
pl.plot(Tzp, Yzp)
pl.show()
[Ttf, Ytf] = ct.impulse_response(Gtf)
pl.plot(Ttf, Ytf)
pl.show()__

**Comment by murrayrm** [comment]
This is a bug.  Created issue #1062 to track.
